dev-python/backports-shutil_which/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+		<email>python@gentoo.org</email>
+		<name>Python</name>
+  </maintainer>
+	<upstream>
```

Comment:
It's nice to use a consistent whitespace method, whether that be tabs or spaces.